firestore-multimodal-genai/PREINSTALL.md

```diff
@@ -67,6 +67,12 @@ This extension will update the state of a document that is being processed withi
 
 Changing the state field of a completed document's status from `COMPLETED` to anything else will retrigger the extension for that document.
 
+### Custom RAG Hook
+
+If you specify a a RAG (Retrieval Augmentation Generation) hook URL during installation, the extension will call this endpoint to obtain additional data for the prompt. You may specify an API key for your hook, which will be passed in the `x-api-key` header. You should also specify input and output fields for the RAG hook, as comma separated lists.
```

Comment:
@pr-Mais  I added docs here, do we need more?

Reply:
Maybe an example?